docs/modules/ROOT/pages/how-tos/update_maintenance.adoc

```diff
@@ -86,6 +86,15 @@ oc --as=cluster-admin get mcp -w
 ----
 ====
 
+. The maintenance of the cluster is only finished once all nodes, including all worker nodes have been upgraded and all `MachineConfigPools` show `Updated=True`.
++
+[IMPORTANT]
+====
+Never leave the cluster in a state with pending node upgrades.
+If the Machine Config operator can't drain a node (eg. because of violating PodDisruptionBudgets) you may have to manually force-drain a node or even manually delete pods that block the node drain.
```

Comment:
I'm surprised Vale didn't complain about the `eg.` but that's possibly because that should be spelled `e.g.`.

```suggestion
If the Machine Config operator can't drain a node (for example because doing so would violate a `PodDisruptionBudget`) you may have to manually force-drain a node or even manually delete pods that block the node drain.
```